cmake/checks.cmake

```diff
@@ -797,8 +753,8 @@ macro(magick_check_env)
   # Check `size_t' size
   check_type_size("size_t" SIZEOF_SIZE_T)
 
-  # Check `ssize_t' size
```

Comment:
This should not be changed I think, and also the block below is needed for me to build on Windows.

Reply:
Should be fixed now.